client/firewall/nftables/manager_linux_test.go

```diff
@@ -0,0 +1,34 @@
+package nftables
+
+import (
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	fw "github.com/netbirdio/netbird/client/firewall"
+)
+
+func TestNftablesManager(t *testing.T) {
+	// just check on the local interface
+	manager, err := Create("lo")
+	require.NoError(t, err)
+
+	ip := net.ParseIP("100.96.0.1")
+
+	rule, err := manager.AddFiltering(
+		ip,
+		fw.ProtocolTCP,
+		&fw.Port{Values: []int{53}},
+		fw.DirectionSrc,
+		fw.ActionDrop,
+		"",
+	)
+	require.NoError(t, err, "failed to add rule")
```

Comment:
Let's check if the rules were added to the system. See example: https://github.com/netbirdio/netbird/blob/main/client/internal/routemanager/nftables_linux_test.go#L162

Reply:
Fixed